src/Command/Email/ConfigurePlatformEmailCommand.php

```diff
@@ -0,0 +1,241 @@
+<?php
+
+namespace Acquia\Cli\Command\Email;
+
+use Acquia\Cli\Command\CommandBase;
+use Acquia\Cli\Exception\AcquiaCliException;
+use Acquia\Cli\Helpers\LoopHelper;
+use AcquiaCloudApi\Connector\Client;
+use AcquiaCloudApi\Endpoints\Applications;
+use AcquiaCloudApi\Endpoints\Environments;
+use AcquiaCloudApi\Response\SubscriptionResponse;
+use React\EventLoop\Loop;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Validator\Constraints\NotBlank;
+use Symfony\Component\Validator\Exception\ValidatorException;
+use Symfony\Component\Validator\Validation;
+
+/**
+ * Class ConfigurePlatformEmailCommand.
+ */
+class ConfigurePlatformEmailCommand extends CommandBase {
+
+  protected static $defaultName = 'email:configure';
+
+  /**
+   * {inheritdoc}.
+   */
+  protected function configure() {
+    $this->setDescription('Configure Platform email for one or more applications')
+      ->addArgument('subscriptionUuid', InputArgument::OPTIONAL)
+      ->addOption('domain', NULL, InputOption::VALUE_REQUIRED)
+      ->setAliases(['ec']);
+  }
+
+  /**
+   * @param \Symfony\Component\Console\Input\InputInterface $input
+   * @param \Symfony\Component\Console\Output\OutputInterface $output
+   *
+   * @return int
+   * @throws \Acquia\Cli\Exception\AcquiaCliException
+   */
+  protected function execute(InputInterface $input, OutputInterface $output) {
+    $this->io->writeln('Welcome to Platform Email setup! This script will walk you through setting up Platform Email, all through the command line and using the Cloud API!');
```

Comment:
Ideally we'd publish a doc page for further info about Platform Email. @klaw772 / @jennifer-babb is this something on your plate yet?

Reply:
@anavarre Yep, we are going to be reaching out to Ashley W. on the Platform Email team to get the ball rolling on this!